examples/Example1_BasicReadings/Example1_BasicReadings.ino

```diff
@@ -12,33 +26,38 @@ uint8_t deviceAddress = kQwiicUltrasonicDefaultAddress; // 0x2F
 
 void setup()
 {
-    // Start serial
-    Serial.begin(115200);
-    Serial.println("Qwiic Ultrasonic Example 1 - Basic Readings");
+  // Start serial
+  Serial.begin(115200);
+  Serial.println("Ultrasonic Distance Sensor Example 1 - Basic Distance Sensing");
 
-    Wire.begin();
+  Wire.begin();
 
-    // Attempt to begin the sensor
-    while (myUltrasonic.begin(deviceAddress) == false)
-    {
-        Serial.println("Ultrasonic sensor not connected, check your wiring and I2C address!");
-        delay(1000);
-    }
+  // Attempt to begin the sensor
+  if (myUltrasonic.begin(deviceAddress) == false)
+  {
+    Serial.println("Ultrasonic sensor not connected, check your wiring and I2C address!");
+    while(1)       
+      ;
```

Comment:
I'm still not a fan of this freezing behavior, since there's a very real possibility of the user missing the one singular print statement, then going down the wrong debugging rabbit hole. Much better IMO to print the error message continuously to basically guarantee that the user sees it. Same comment for all other examples.

Reply:
I completely agree and wouldn't have changed it, had I understood your reasoning.